chromium_src/components/sync/base/model_type.cc

```diff
@@ -14,6 +14,7 @@ ModelTypeSet EncryptableUserTypes() {
   ModelTypeSet encryptable_user_types = EncryptableUserTypes_ChromiumImpl();
   // Brave sync has encryption setup ready when sync chain created
   encryptable_user_types.Put(DEVICE_INFO);
+  encryptable_user_types.Put(HISTORY);
```

Comment:
don't we need to remove `HISTORY_DELETE_DIRECTIVES` from this list?

Reply:
[EncryptableUserTypes](https://source.chromium.org/chromium/chromium/src/+/main:components/sync/base/model_type.cc;l=583?q=EncryptableUserTypes&ss=chromium%2Fchromium%2Fsrc)
already has 
```
encryptable_user_types.Remove(HISTORY_DELETE_DIRECTIVES);
```
so I didn't put it here.